qemu/tests/cfg/virtio_fs_support_win_fs.cfg

```diff
@@ -0,0 +1,50 @@
+- virtio_fs_support_win_fs:
+    type = virtio_fs_support_win_fs
+    only Windows
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    kill_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem_devs = mem1
```

Comment:
For this specific test, do you have to use Numa with virtio_fs? If not, I suggest we use it follow the new method provides by https://github.com/avocado-framework/avocado-vt/pull/3666

If we have to use Numa with virtio_fs in this specific test scenario, you could add this for s390x:
s390, s390x:
        required_qemu = [5.2.0,)
        vm_mem_share = yes
        mem_devs =
        guest_numa_nodes =
        pre_command_noncritical = yes
        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
        vm_mem_backend = memory-backend-file
        vm_mem_backend_path = /dev/shm

Reply:
> For this specific test, do you have to use Numa with virtio_fs? If not, I suggest we use it follow the new method provides by [avocado-framework/avocado-vt#3666](https://github.com/avocado-framework/avocado-vt/pull/3666): vm_mem_backend = memory-backend-file vm_mem_backend_path = /dev/shm and remove the second mem_devs and numa related cfg
> 
> If we have to use Numa with virtio_fs in this specific test scenario, you could add this for s390x: s390, s390x: required_qemu = [5.2.0,) vm_mem_share = yes mem_devs = guest_numa_nodes = vm_mem_backend = memory-backend-file vm_mem_backend_path = /dev/shm

Hello @xiagao , what's your opinion at here? What should this case take?

Reply:
I think with numa or not are okay for this case.

Reply:
> I think with numa or not are okay for this case.

Okay, nothing change at here. 
Thanks!

Reply:
@nickzhq for s390x, please update as this comment. If with or without numa are taking, please add the s390x part.